internal/pkg/featureflags/feature_flags.go

```diff
@@ -57,16 +57,25 @@ type launchDarklyManager struct {
 	version               *version.Version
 }
 
-func Init(version *version.Version, isTest, isDisabledConfig bool) {
+func Init(cfg *v1.Config) {
+	var platformName string
+	if !cfg.IsTest {
+		currentContext, err := cfg.FindContext(cfg.CurrentContext)
+		if err != nil {
+			log.CliLogger.Warnf("Current context %s not found", cfg.CurrentContext)
+		} else {
+			platformName = currentContext.PlatformName
+		}
```

Comment:
can't we just do `cfg.Context.GetPlatformName()` here?

Reply:
`cfg.Context()` and then calling another method isn't panic-safe if the user edits their config.